packages/fxa-react/components/Head/en.ftl

```diff
@@ -1,8 +1,15 @@
 ## FxA React - Strings shared between multiple FxA products for application page title
 
 app-default-title = { -product-firefox-accounts }
+# This string is used as the default title for pages, displayed in the browser tab.
+app-default-title-2 = { -product-mozilla-accounts }
 # This string is used as the title of the page.
 # Variables:
 #   $title (String) - the name of the current page
 #                      (for example: "Two-step authentication")
 app-page-title = { $title } | { -product-firefox-accounts }
+# This string is used as the title of the page, displayed in the browser tab.
+# Variables:
+#   $title (String) - the name of the current page
+#                      (for example: "Two-step authentication")
+app-page-title-2 = { $title } | { -product-mozilla-accounts }
```

Comment:
@bcolsson I just realized I already made these changes [here](https://github.com/mozilla/fxa/pull/15800/files#diff-7047ed172a0622ec08c51c2302c6138a85b99f728fe726ca99d53981c51cf77dR14) but had double checked on the page title and got confirmation yesterday and apparently made the change again on the wrong branch heh.

Is the "displayed in the browser tab" helpful at all in your opinion? If so I'll transfer my comments over to #15800, merge it, and revert changes here, otherwise I'll just merge the other now and revert my changes here.

Reply:
I'd say "displayed in the browser tab" would be nice to have in the comment, since it's more easily understood exactly where the string is shown.